src/lib/expansion/expansion-panel-header.scss

```diff
@@ -43,16 +43,16 @@
   flex-grow: 2;
 }
 
-/**
- * Creates the expansion indicator arrow. Done using ::after rather than having
- * additional nodes in the template.
- */
-.mat-expansion-indicator::after {
+.mat-expansion-indicator-container {
+  margin-bottom: 2.83px;
```

Comment:
These is weirdly specific- is there any way this can be calculated or set as the result of other known values?